cryptographic_estimators/__init__.py

```diff
@@ -12,4 +12,14 @@
 from . import DummyEstimator
 from . import MREstimator
 from . import UOVEstimator
-from . import MAYOEstimator
\ No newline at end of file
+from . import MAYOEstimator
```

Comment:
or even better: `CryptographicEstimator/MQEstimator/monomial.py`

Reply:
I decided to put it there because it's a very tricky and non-standard setup requirement to work with power series (in comparison with Sage, for example). This way, any new estimator that will use flint power series can be implemented seamlessly.

Another reason is to avoid bugs/test errors that can be VERY hard to debug, and can be produced by incompatible overrides of this parameter. Think in situations like this: you are implementing series 3 that uses series1 with cap at 10, and series2 with cap at 20. Even if you set your cap in series 3 as 30, if you make subsequent calls to either series1 or series2, you will change the cap you previously set; potentially breaking your calculations.

Reply:
@FloydZ  `UOV` and `MAYO` estimators also use PowerSeries. Maybe MREstimator will need it in  the future as well.

Reply:
jeah than its fine to keep it here